lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -292,11 +337,15 @@ int main(int argc, char* argv[]) {
     proxy_service.wait_for_requests();  // block here instead of on server
     proxy_service.stop();               // stop queue after server shuts down
   });
-
+  MLOG(MINFO) << "start test server";
```

Comment:
Is this a temporal log that was left forgotten?
If not, this should be `MDEBUG`

Reply:
removed

Reply:
Changed  as MDEBUG